### DISCUSSION TITLE
sementical error

### DISCUSSION THREAD

**Discussion by dialloxavi06** [symfony/symfony#54658] [Q&A]
I encountered an issue while running php bin/console make:migration in my Symfony project. The error message I received is as follows:

vbnet

[Semantical Error] The class "Doctrine\ORM\Mapping\MappedSuperclass" is not annotated with @Annotation.   
Are you sure this class can be used as an annotation? If so, then you need to add @Annotation to the doc comment of "Doctrine\ORM\Mapping\MappedSuperclass". If it is indeed not an annotation, then you need to add @IgnoreAnnotation("ORM\MappedSuperclass") to the doc comment of the class Gedmo\Translatable\Entity\MappedSuperclass\AbstractTranslation.

It seems like there's an issue with annotations in my project. Can someone please help me understand why I'm encountering this error and how I can resolve it? Thank you!

**Comment by dialloxavi06** [comment]
I'm currently working on a Symfony project and trying to utilize the stof/doctrine-extensions-bundle. I've configured all the necessary files according to the documentation, including config/packages/stof_doctrine_extensions.yaml and config/packages/doctrine.yaml. However, when I run php bin/console make:migration, I encounter the following error:

[Semantical Error] The class "Doctrine\ORM\Mapping\MappedSuperclass" is not annotated with @Annotation.   
Are you sure this class can be used as an annotation? If so, then you need to add @Annotation to the doc comment of "Doctrine\ORM\Mapping\MappedSuperclass". If it is indeed not an annotation, then you need to add @IgnoreAnnotation("ORM\MappedSuperclass") to the doc comment of the class Gedmo\Translatable\Entity\MappedSuperclass\AbstractTranslation.

I'm unsure why this error is occurring, especially since I've followed the bundle's documentation for configuration. Could someone please provide insight into what might be causing this issue and how I can resolve it? Any assistance would be greatly appreciated. Thank you!

**Reply by mbabker** [reply]
You are trying to use ORM 3.x, the `gedmo/doctrine-extensions` library is not yet compatible with that version of the ORM.  Downgrade to ORM 2.19.

**Reply by dialloxavi06** [reply]
thank you, it work now

**Comment by ghost** [comment]
Dude, please clean up the title. Are kidding us?
